src/serverconfig.example.json

```diff
@@ -86,16 +87,20 @@
         "google": {
             "clientId": "",
             "clientSecret": "",
-            "batchSize": 50,
+            "batchSize": 30,
```

Comment:
Why was this changed to 30? I think it should be default of 50 for all handlers.

Reply:
It was for test, and reverted